### ISSUE TITLE
Add `lint.sh` tool to make linting easier.

### ISSUE THREAD

**Issue by brownhead** [brownhead/phial#40]
It would run the lint checks, but also give instructions on installing them (or just install them automatically, but that seems a little aggressive). I could also put it in a make file.

**Comment by brownhead** [comment]
Resuming work on this. Did not take notes -_-. Trying to pick up the pieces and see what I was doing here...

**Comment by brownhead** [comment]
Meh, been too long since I've managed a many projects at once, my good habits are getting rusty.

**Comment by brownhead** [comment]
Alright, so I was writing something to hit the flake8 API but it doesn't seem to do the same things as calling the binary directly (unless I actually invoked that same path through main). Since the binary already has extensions if I want to add onto it, I don't see a win going this route, especially since I doubt I'll even add extensions. I imagine I started doing that because I wanted to indulge a bit during a late night.

Anyways, going to go the simpler route.
